### ISSUE TITLE
Implement a learning rate schedule

### ISSUE THREAD

**Issue by pokey** [chaosparrot/parrot.py#18]
Currently, it appears learning rate is constant.  My loss started to really slow down; I think dropping it could help a lot, but there is no schedule, and due to #17, I couldn't just restart it with a smaller LR

**Comment by ym-han** [comment]
As a related quick note to self: In addition to implementing some sort of learning rate scheduler, we should also add a learning rate finder. Lightning seems to ship with one: https://pytorch-lightning.readthedocs.io/en/1.4.5/advanced/lr_finder.html

**Comment by pokey** [comment]
yes was thinking a lr range test would be a good idea; forgot that was automated in lighning.  good stuff
